Documentation/platforms/arm/stm32h7/boards/portenta-h7/index.rst

```diff
@@ -0,0 +1,80 @@
+===================
+Arduino Portenta H7
+===================
+
+This page discusses issues unique to NuttX configurations for the
```

Comment:
@raiden00pl please attach the png image of the board.

Reply:
https://github.com/apache/nuttx/pull/17298#issuecomment-3506753849